src/adapters/pg/store.rs

```diff
@@ -78,10 +77,14 @@ where
             .lazy_query(&trans, &params, 1000)
             .expect("Query")
             .map(|row| {
-                let json: JsonValue = row.get("data");
-                let evt: E = from_value(json).expect("Decode");
+                let id: Uuid = row.get("id");
+                let data_json: JsonValue = row.get("data");
+                let context_json: JsonValue = row.get("context");
```

Comment:
It would be nice to define the basic characteristics of the subject as a struct. Let's leave it for another PR